cpu/cc26x0/include/cc26x0_prcm.h

```diff
@@ -67,13 +67,13 @@ typedef struct {
 #define DDI_0_OSC_CTL0_ACLK_TDC_SRC_SEL_RCOSC_HF    0x000 /* 48MHz */
 #define DDI_0_OSC_CTL0_ACLK_TDC_SRC_SEL_RCOSC_LF    0x080 /* 48MHz */
 #define DDI_0_OSC_CTL0_ACLK_TDC_SRC_SEL_XOSC_HF     0x100 /* 24MHz */
-#define DDI_0_OSC_CTL0_CLK_LOSS_EN                  0x200 /* enable clock loss detection */
-#define DDI_0_OSC_CTL0_XOSC_LF_DIG_BYPASS           0x400 /* bypass XOSC_LF and use digital input clock from AON foor xosx_lf (precuations in datasheet) */
-#define DDI_0_OSC_CTL0_XOSC_HF_POWER_MODE           0x800
-#define DDI_0_OSC_CTL0_RCOSC_LF_TRIMMED             0x1000
-#define DDI_0_OSC_CTL0_ALLOW_SCLK_HF_SWITCHING      0x10000
-#define DDI_0_OSC_CTL0_FORCE_KICKSTART_EN           0x400000
-#define DDI_0_OSC_CTL0_DOUBLER_RESET_DURATION       0x2000000
+//#define DDI_0_OSC_CTL0_CLK_LOSS_EN                  0x200 /* enable clock loss detection */
+//#define DDI_0_OSC_CTL0_XOSC_LF_DIG_BYPASS           0x400 /* bypass XOSC_LF and use digital input clock from AON foor xosx_lf (precuations in datasheet) */
+//#define DDI_0_OSC_CTL0_XOSC_HF_POWER_MODE           0x800
+//#define DDI_0_OSC_CTL0_RCOSC_LF_TRIMMED             0x1000
+//#define DDI_0_OSC_CTL0_ALLOW_SCLK_HF_SWITCHING      0x10000
+//#define DDI_0_OSC_CTL0_FORCE_KICKSTART_EN           0x400000
+//#define DDI_0_OSC_CTL0_DOUBLER_RESET_DURATION       0x2000000
```

Comment:
Did you comment this out because it's also in the vendor code?
If so, just remove it instead of commenting it out.